<?xml version="1.0"?>

<launch>
    <arg name="visual_mapping_config" default="dino_vlad.yaml"/>

    <node pkg="physics_atv_visual_mapping" type="dino_localmapping.py" name="visual_localmapping" output="screen" required="true">
        <param name="config_fp" value="$(find physics_atv_visual_mapping)/config/ros/$(arg visual_mapping_config)"/>
    </node>

    <node pkg="tf" type="static_transform_publisher" name="multisense_to_base" args="-.093 0.103674 -0.127 0.0256 0.2185394 0.0063011 vehicle multisense/head 100"/>

    <node pkg="pointcloud_colorization" type="multisense_intrinsics_pub.py" name="multisense_intrinsics_pub"/>

    <node pkg="physics_atv_visual_mapping" type="cloud_repub.py" name="lidar_stitching"></node>

</launch>
